### DISCUSSION TITLE
`useFieldArray` without `defaultValue`

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#5645] [General]
One of the things that bother us a lot is an inconsistency between normal input and inputs within `useFieldArray`, 

> defaultValue must be set for all inputs. Supplied defaultValues in the useForm hook will prepare the fields object with default value. 

We are trying to overcome such behavior, with the following PR: https://github.com/react-hook-form/react-hook-form/pull/5636

The current PR passes all our unit tests and automation, but we are hesitant to release such a chance before everything is ticked off. So we are hoping the community can test out this update before we release the update.

You can start and help to test with the following beta version:

`react-hook-form@7.8.9-beta.1`

**Comment by Evalon** [comment]
This fix can't come soon enough!
Thank you, hope this is going to be released soon! 👍 
(Tried beta for my **simple** cases - everything works as expected.)

**Reply by bluebill1049** [reply]
Thank you for the testing!

**Comment by iKrushYou** [comment]
Does this still need to be tested? I noticed we're many versions past

**Reply by bluebill1049** [reply]
I think we are good without the `defaultValue` 👍
